src/customers/account-dashboard-gift-cards.md

```diff
@@ -7,3 +7,34 @@ The Gift Card section of your account dashboard can be used to check the balance
 
 ![]({% link images/images-ee/account-dashboard-gift-card.png %}){: .zoom}
 _Gift Card_{:.ee-only}
```

Comment:
I know this isn't yours but let's clean this one up too. Since the whole page is Commerce only, it is not needed here. 

```suggestion
_Gift Card_
```